pkgs/cupertino_http/example/lib/get_books.dart

```diff
@@ -0,0 +1,38 @@
+import 'dart:convert';
```

Comment:
Don't you only see `main.dart` on pub (i.e. https://pub.dev/packages/cupertino_http/example) so the network code will disappear from the example.

Reply:
Good point, moved the JSON code to book.dart and then the networking code back to main.dart.